docs/src/js/base/main.js

```diff
@@ -342,3 +342,38 @@ document.querySelectorAll('.switch:not(.switch--form)').forEach(function(widgetE
         console.log(e.type, e.detail);
     });
 });
+
+// TOAST
+document.querySelectorAll('.toast-button').forEach(function(openToastButton) {
+    const toastElement = openToastButton.nextElementSibling;
+    if (!toastElement.classList.contains('toast')) {
+        console.warn(`Unexpected element ${toastElement.tagName} after show toast button! Expected toast element.`);
+        return;
+    }
+
+    const isTransitionToast = toastElement.classList.contains('toast--transition');
+    let closeToastButton;
+
+    function handleToastClose() {
+        toastElement.setAttribute('hidden', '');
+        if (isTransitionToast) {
+            toastElement.classList.remove('toast--show');
```

Comment:
Oh I just noticed, you did not use the `transition` helper as in `dialog`. We should use that instead of trying to handle it on our own. 
Again, please try to follow that as closely as you can.

Reply:
@agliga We talked about this on Slack and decided to not use `transition.js` ~

```
Just to finalize, no need to use window or transition javascript. try to remove animation and use transition css property right?
```

Reply:
Yes. But trasition.js does that already for you. 
So better to use that than to reinvet the wheel. 
However I'm fine with merging this as is as ian said and taking a look later at it.

Reply:
I think it might make sense for consistency sake to use window, given the direction I want to go (https://github.com/eBay/skin/issues/1082) where any dialog can potentially be modal (with mask) or modeless (without mask). The window element gives us that extra hook for animations & transitions I believe. I don't mind taking care of this in a follow up PR, it will help jog my memory of how the dialog animations work (it's been a long time).

Reply:
And also it is necessary to use window for at least eBay ui since we will reuse the dialog common component with toast.

Reply:
@ianmcburnie Sure :)